### PR TITLE
[BUGFIX beta] use import type in helpers/index.ts :: typechecking in new apps fails atm

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
Resolves https://github.com/ember-cli/ember-cli/issues/10401

### CODE REVIEW

File: blueprints/app/files/tests/helpers/index.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import {
   setupApplicationTest as upstreamSetupApplicationTest,
   setupRenderingTest as upstreamSetupRenderingTest,
   setupTest as upstreamSetupTest,
-  SetupTestOptions,
+  type SetupTestOptions,
```
Comment (NullVoxPopuli) [inline blueprints/app/files/tests/helpers/index.ts:5]: `@tsconfig/ember` uses `verbatimModuleSyntax` which requires type imports to be declared

**Comment by kategengler** [comment]
If you want this to go to release, this should target branch `release`. CLI merges to the earliest we want to release and merges forward to other branches.

**Comment by NullVoxPopuli** [comment]
@kategengler here is one targeting `release`: https://github.com/ember-cli/ember-cli/pull/10402

**Comment by kategengler** [comment]
Closing in favor of #10402 (which will get merged-forward)

**Pull request closed without merging**